.config/guardian/TSAConfig.gdntsa

```diff
@@ -0,0 +1,15 @@
+{
+    "codebaseName": "VSCode-CSharp-GitHub",
+    "notificationAliases": [
+        "mlinfraswat@microsoft.com"
+    ],
+    "codebaseAdmins": [
+        "REDMOND\\jaredpar",
```

Comment:
Probably replace Jared with Phil

Reply:
Unless this is automatically generated

Reply:
nah, just copied from Roslyn. Will updated.

Reply:
Updated.